docs/pages/world/modules.mdx

```diff
@@ -55,7 +55,7 @@ keyTuple[0] = "bob";
 bool memory isPlayer = hasKey(world, PlayerTableId, keyTuple);
 ```
 
-Internally, it works by installing a [hook](http://localhost:3000/store/advanced-features#storage-hooks) that maintains:
+Internally, it works by installing a [hook](https://v2.mud.dev/store/advanced-features#storage-hooks) that maintains:
```

Comment:
Can you use a relative path instead? It will work both on localhost and when published:
`/store/advanced-features#storage-hooks` will work

Reply:
ooh, duh. thanks - means we don't point to `v2.` either.